meal_planner/database.py

```
@@ -1,14 +1,49 @@
+and provides functions for initializing the database schema.
+"""
+
+from sqlmodel import Session, SQLModel, create_engine, select
 
 from meal_planner.config import CONTAINER_MAIN_DATABASE_URL
 
-engine = create_engine(
-    CONTAINER_MAIN_DATABASE_URL, echo=True, connect_args={"check_same_thread": False}
-)
+# Don't check validity for SQLite
+engine = create_engine(CONTAINER_MAIN_DATABASE_URL, connect_args={"check_same_thread": False})
```

Ruff (0.11.9)

Line too long (94 > 88)

(E501)